python/src/scipp/plotting/model3d.py

```diff
@@ -121,7 +121,7 @@ def update_data(self, slices, mask_info):
                                  variances=np.zeros(shape),
                                  dtype=data_slice.data.dtype,
                                  unit=sc.units.one),
-                masks=data_slice.masks)
+                masks=data_slice.masks.to_dict())
```

Comment:
This now appears to be the only place where `to_dict` is used? Maybe we should avoid adding it to scipp objects everywhere (in `__init__.py` above) and just to this locally?

Reply:
hmm i found 3 places where it is used:
```
(base) nvaytet@this:~/code/scipp/source/python/src/scipp/plotting$ gr to_dict
__init__.py:    if hasattr(meta, "to_dict"):
__init__.py:        meta = meta.to_dict()
model3d.py:                masks=data_slice.masks.to_dict())
model.py:                masks=item["array"].masks.to_dict())
```

Reply:
I'm thinking of maybe renaming this to `asdict` to be more like `astype`? Thoughts?

Reply:
Make it a free function and keep it plotting internal?

Reply:
In `plotting/tools.py` ?

Reply:
I don't care much, the point is that we have to be careful about adding new functions to the public API, since sooner or later these will become "hard" to remove or modify since many users will start depending on them.